src/charts/Bar.js

```diff
@@ -319,10 +319,38 @@ class Bar {
     const graphics = new Graphics(this.ctx)
 
     if (!lineFill) {
+     // if user provided a function in colors, we need to eval here
+     // Note: the position of this function logic (ex. stroke: { colors: ["",function(){}] }) i.e array index 1 depicts the realIndex/seriesIndex.
+      function fetchColor(i) {
+        const exp = w.config.stroke.colors
+        let c
+        if (Array.isArray(exp) && exp.length > 0) {
+          c = exp[i]
+          if (!c) c = ''
+          if (typeof c === 'function') {
+            return c({
+              value: w.globals.axisCharts
```

Comment:
w.globals.axisCharts will always be true for bar chart, so no point in having this condition here.

Reply:
done